src/include/common/fe_memutils.h

```diff
@@ -29,11 +29,10 @@ extern void *pg_malloc_extended(size_t size, int flags);
 extern void *pg_realloc(void *pointer, size_t size);
 extern void pg_free(void *pointer);
 
-<<<<<<< HEAD
 #ifdef EXTRA_DYNAMIC_MEMORY_DEBUG
 #include "utils/palloc_memory_debug_undef.h"
 #endif
-=======
+
```

Comment:
Is this empty line needed?

Reply:
I think it wouldn't be very nice to stick a comment to the endif.